ajax/libs/Sharrre/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Sharrre",
+  "filename": "jquery.sharrre.min.js",
+  "version": "2.0.1",
+  "description": "Make your sharing widget! Sharrre is a jQuery plugin that allows you to create nice widgets sharing for Facebook, Twitter, Google Plus (with PHP script) and more.",
+  "author": "Julien Hany",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Julienh/Sharrre.git"
+  },
+  "homepage": "http://sharrre.com",
+  "keywords": [
+    "jQuery",
+    "plugin",
+    "share",
+    "widget"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Julienh/Sharrre.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(test.css)"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "js/jquery.sharrre*"
```

Comment:
I think maybe we still need the files which under `js/platform` as the [README](https://github.com/Julienh/Sharrre#installing) said.
What do you think?
Thank you.